packages/rke2-multus/charts/templates/configMap.yaml

```diff
@@ -15,7 +15,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ .Release.Name }}-{{ .Chart.Version }}-config
+  name: {{ .Release.Name }}-{{ .Chart.Name }}-{{ .Chart.Version }}-config
```

Comment:
This change will create a ConfigMap with `rke2-multus` listed twice:
```
rke2-multus-rke2-multus-v4.1.001-config
```

Instead of modifying the ConfigMap, could you remove the `Chart.Name` from the DaemonSet instead?
https://github.com/rancher/rke2-charts/blob/af7c7c9a24c746866ba60717bc20ca21cf406b21/packages/rke2-multus/charts/templates/daemonSet.yaml#L133

Reply:
Sure

Reply:
Thanks!